src/test/java/org/interledger/InterledgerAddressTest.java

```diff
@@ -123,4 +135,9 @@ public void test_address_hashcode() {
     assertThat(addressPrefix1.toString().equals(addressPrefix3.toString()), is(false));
   }
 
+  @Test
+  public void testToString() throws Exception {
+    assertThat(InterledgerAddress.of("g.foo.bob").getValue().toString(), is("g.foo.bob"));
```

Comment:
This test passes, but is incorrect - I'll fix this in the next PR where I update the toString of `InterledgerAddress`.